README.md

```diff
@@ -50,6 +50,19 @@ Run:
   $ bin/setup
   ```
 
+
+### Add Local Environment Variables
+Run `cp .sample.env .env` to create a new .env file
```

Comment:
The `bin/setup` script [already does this](https://github.com/thoughtbot/constable/blob/master/bin/setup#L6-L10), so I'm not sure if it makes sense to call out explicitly. What do you think about removing this first line?

Reply:
sure!